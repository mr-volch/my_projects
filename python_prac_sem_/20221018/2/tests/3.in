:exp x y z cos((x+y)**z)
exp 3 1 2
:len s d a len(s+d)-len(a)
len "ЖЖЖ" "AAAА" "С"
exp 0.5 1.5 3
quit "{} !? {}"
