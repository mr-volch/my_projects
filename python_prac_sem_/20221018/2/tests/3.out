-0.9576594803233847
6
-0.14550003380861354
3 !? 6
